src/wallet.cpp

```diff
@@ -2611,3 +2611,53 @@ void CWallet::GetKeyBirthTimes(std::map<CKeyID, int64_t> &mapKeyBirth) const {
         mapKeyBirth[it->first] = it->second->nTime - 7200; // block times can be 2h off
 }
 
+MinedType GenerateType(const uint256& tx, unsigned int vout)
```

Comment:
Can this function be renamed so it does not suggest that the Type is going to be Generated?

Reply:
I guess so. This was Paul's choice.

Reply:
What would you like it to be called?

Reply:
getGenerateType, getTypeOfGenerated, TypeOfGeneration, (get)GenerationTxType

Reply:
Changed it to getGeneratedType which is more descriptive and not misleading.

Reply:
I don't think you pushed that change. Also, uppercase 'G' :)

Reply:
Fixed.